nilearn/glm/tests/test_bids.py

```diff
@@ -0,0 +1,177 @@
+"""Test the first level model on BIDS datasets."""
```

Comment:
Extracted all the tests for first level BIDS into a separate file to make it easier to navigate.
Not sure if nilearn follows the "convention" -> all tests must for a given module must be in a single file.

Reply:
Yes, we strive to follow that convention.

Reply:
ok brought back everything into the original file